sycl/plugins/unified_runtime/ur/adapters/cuda/common.cpp

```diff
@@ -98,6 +98,20 @@ thread_local char ErrorMessage[MaxMessageSize];
   ErrorMessageCode = ErrorCode;
 }
 
+void setPluginSpecificMessage(CUresult cu_res) {
```

Comment:
TODO: since this cannot yet be used due to the recent bug: https://github.com/intel/llvm/issues/10066 , I could remove it from the PR. However it will be added later anyway so I see no harm in including it here.